pkg/templates/images/images.go

```diff
@@ -301,40 +301,47 @@ func optionalResources() map[Resource]map[string]string {
 
 		// OpenStack CCM
 		OpenstackCCM: {
-			"1.24.x":    "docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.24.5",
```

Comment:
We technically still support Kubernetes 1.24.

Reply:
1.24.x restored for all openstack images